aiida-registry-app/src/App.jsx

```diff
@@ -1,406 +1,101 @@
-                    <li key={ep_name}>
-                      <code style={{fontSize: '20px'}}>{ep_name} </code>
-                      {typeof ep_module === "string" ? (
-                      <div className="classbox">
-                        class
-                        <span className="tooltiptext"> {ep_module}</span>
-                      </div>
-                      ) : (
-                        <EntryPoints entryPoints={ep_module} />
-                      )}
-                    </li>
-                  ))}
-                </ul>
-              </div>
-            ))
-          ) : (
-            <p>No entry points defined for this plugin.</p>
-          )}
-        </>
-      ) : (
-        <div id="description">
-          <p>
-            Detailed information for this package could not be obtained. Ask the
-            plugin author to add a <code>setup.json</code> file to the plugin
-            source code.
-          </p>
-        </div>
-      )}
-    </div>
-    </>
-  );
+        <footer  className="footer">
+          <hr />
+          The official <a href="http://aiidateam.github.io/aiida-registry">registry</a> of <a href="http://www.aiida.net">AiiDA</a> plugins.
+          <br />
+          This work is supported by the <a href="http://nccr-marvel.ch" target="_blank">MARVEL National Centre for Competence in Research</a> funded by the <a href="http://www.snf.ch/en" target="_blank">Swiss National Science Foundation</a>, as well as by the <a href="http://www.max-centre.eu" target="_blank">MaX European Centre of Excellence</a> funded by the Horizon 2020 EINFRA-5 program, Grant No. 676598.
+          <br /><br />
+
+          <div style={{ textAlign: 'center' }}>
+            <img src={MARVEL} height="70px" />&nbsp;&nbsp;&nbsp;&nbsp;<img src={MaX} height="70px" />
+          </div>
+        </footer>
+  )
 }
 
-const EntryPoints = ({entryPoints}) => {
-  return (
-
-<div>
-<table>
-    <tbody>
-        <tr>
-            <th>Class</th>
-            <td><code>{entryPoints.class}</code></td>
-        </tr>
-    </tbody>
-</table>
-<table>
-<tr>
-        <th>Description</th>
-    </tr>
-      {entryPoints.description.map((description) => (
-    <tr className='ep_description'>
-        <Markdown>{description.trim()}</Markdown>
-    </tr>
-      ))}
-</table>
-
-<table>
-
-    <tr>
-        <th>Inputs</th>
-        <th>Required</th>
-        <th>Valid Types</th>
-        <th>Description</th>
-    </tr>
-    <Specs spec={entryPoints.spec.inputs} />
-
-    <tr>
-        <th>Outputs</th>
-        <th>Required</th>
-        <th>Valid Types</th>
-        <th>Description</th>
-    </tr>
 
-    <Specs spec={entryPoints.spec.outputs} />
+/**
+ * DetailsContainer component displays the details of a specific plugin identified by the key parameter.
+ * It renders the Details component on the left side and the Sidebar component on the right side.
+ *
+ * @component
+ * @example
```

Comment:
What is this `@example`?

Reply:
it is used to show how the component should be rendered, it is good for complex components with parameters. but it doesn't add much for simple components like `<DetailsContainer />` so I'll delete it.